circle.yml

```diff
@@ -2,7 +2,7 @@ machine:
   environment:
     DOCKER_IMAGE_NAME: prom/node-exporter
     QUAY_IMAGE_NAME: quay.io/prometheus/node-exporter
-    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.6-base
+    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.7-base
```

Comment:
Came across this too and created https://github.com/prometheus/golang-builder/issues/19 since I couldn't figure it out. Also looks like each variant has their own slightly different copy of the build script etc.